test/e2e/scheduler_predicates.go

```diff
@@ -169,6 +169,22 @@ var _ = Describe("SchedulerPredicates [Serial]", func() {
 
 		err = waitForPodsRunningReady(api.NamespaceSystem, systemPodsNo, podReadyBeforeTimeout)
 		Expect(err).NotTo(HaveOccurred())
+
+		for _, node := range nodeList.Items {
+			Logf("\nLogging pods the kubelet thinks is on node %v before test", node.Name)
+			podList, err := GetKubeletPods(c, node.Name)
```

Comment:
This function hits the /runningpods endpoint to get the pods that are running on the node (i.e., have at least one alive container shown in `docker ps`). kubelet admits pods by looking at the "desired list of pods", which is exposed at /pods. In your case, you will want to hit the /pods endpoint.